components/brave_wallet_ui/common/async/wallet_async_handler.ts

```diff
@@ -41,19 +41,24 @@ async function refreshWalletInfo (store: Store) {
   const apiProxy = await getAPIProxy()
   const walletHandler = apiProxy.walletHandler
   const ethJsonRpcController = apiProxy.ethJsonRpcController
+  const assetPriceController = apiProxy.assetRatioController
   const result = await walletHandler.getWalletInfo()
   store.dispatch(WalletActions.initialized(result))
   const network = await ethJsonRpcController.getNetwork()
   store.dispatch(WalletActions.setNetwork(network.network))
 
   // Update balances
   const state = getWalletState(store)
-
+  const ethPrice = await assetPriceController.getPrice(['eth'], ['usd', 'btc'])
```

Comment:
Why do we fetch btc price here, doesn't seem to be used anywhere?

Reply:
Removed!